GroupMeClientApi/Models/ContactsList.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace GroupMeClientApi.Models
+{
+    /// <summary>
+    /// <see cref="ContactsList"/> provides a list of <see cref="Contact"/>s along with additional status information.
+    /// </summary>
+    public class ContactsList
```

Comment:
Since we don't see to expose this outside the Api anywhere and the `IEnumerable<Contact>` and `List<Contact>` wrap it in `GroupMeClient`, I think this can safely be private. Or at least internal.